contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,133 @@
+
+If a specific user requested the feature or has a problem potentially solved by it, we might roll it out to them with the expectation that it is recently built. Feature flags enable us to adjust the rollout and do this fast. We communicate with the user through this process and ideally receive feedback that helps us improve it.
+
+Sampling got its first real users through the support hero tagging Yakko in places they felt sampling might help out. This helped improve the experience for customers without completing the feature. When these users were positive about their experience, he knew the feature was successful so far and continued the rollout. 
+
+![Message](../images/blog/measuring-feature-success/message.png)
+
+A feature is successful at this stage if users use it and the qualitative feedback we receive from them is positive. A common goal at this stage is "5 happy teams using it."
+
+### 3. Launch
+
+When we have positive feedback and usage in beta, and the feature reaches the point we are ready to release. The flag rolls out further, eventually to 100% of users.
+
+Yakko continued to update the simple UI for sampling (it changed four times) to ultimately land on something he was happy with. He instrumented tracking for the feature’s usage and rolled it out to 25% of all organizations in PostHog.
+
+![Sampling UI](../images/blog/measuring-feature-success/sampling.png)
+
+At this stage, success is significant usage by the target user. What this portion is depends on the usage of similar features. At PostHog, this looks like:
+
+- 10-12% of our daily active users use individual products (like product analytics or session recordings)
+- 15-40% of daily active product users use a large features (like flag payloads or funnels) within those products.
+- 4-12% of daily active product users use a small features (like sampling, replay network tab, or dashboard templates) within those products.
+
+The feature should also be issue free and work as intended. Bug reports or confusion on how to use the feature are bad signs.
+
+To evaluate success after launch, we watch session replays and track metrics that match specific criteria we care about. For example, we might look at the session recordings for high ICP, paid users in the 99th percentile of feature usage to see how power users are utilizing the product. We also might create a dashboard to show usage and the feature’s impact (such as average insight load time).
+
+In Yakko’s case, he: 
+
+- created a key metrics dashboard that he would dig into a few times a day.
+- watched session replays of the feature to see if it was having the intended impact or causing any confusion (he fixed a few visual regressions as a result of this).
```

Comment:
Could be a good blog post in itself. Do we already a blog on "best practices on how to use session replays to gain insights" ?

Reply:
found it: https://posthog.com/tutorials/explore-insights-session-recordings
Maybe worth linking to it here?